vuejs/src/components/TVChartContainer.vue

```diff
@@ -65,12 +61,13 @@ export default {
   },
   tvWidget: null,
   mounted() {
+    const container = document.getElementById('tv-chart-container')
```

Comment:
It seems that we can use vue's refs for that https://blog.logrocket.com/vue-refs-accessing-dom-elements/ 🤔